projects/cds/src/profiletag/model/profile-tag.model.ts

```diff
@@ -38,24 +38,86 @@ interface ProfileTagCart {
   cart: Cart;
 }
 
-export interface ProfileTagEvent {
+export interface ProfileTagPushEvent {
   name: string;
-  data?: any;
+  data?: {
+    segments?: string[];
+    actions?: PersonalizationAction[];
+    [x: string]: any;
+  };
 }
 
-export class NavigatedPushEvent implements ProfileTagEvent {
+export class NavigatedPushEvent implements ProfileTagPushEvent {
   name = 'Navigated';
+  data;
```

Comment:
Can we change all the instances of `data;` to `data: any`?

Reply:
Changed them in profile-tag.model.ts